themes/default/assets/js/main.js

```diff
@@ -305,6 +317,50 @@ var initializers = {
     }
 };
 
+function isSessionStorageAccessible() {
+    try {
+        sessionStorage
+    } catch(e) {
+       return false;
+    }
+    return true;
+}
+
+class DataStore {
+    constructor(data) {
+        this.data = data;
+    }
+
+    insertItem(key, value) {
+        console.log(`Inserting ${key} : ${value} into data store.`);
```

Comment:
remove all `console.log`s please. they are useful only for debugging, and in this case of not much use.

we don't want the browser's console to be clutterred.